src/main/java/bio/terra/service/dataset/StorageResourceDao.java

```diff
@@ -96,7 +105,8 @@ public void createStorageAttributes(List<StorageResource> storageResources, UUID
             valuesList.add(String.format("(:dataset_id, :%s, :%s, :%s)",
                 regionParam, cloudResourceParam, platformParam));
 
-            params.addValue(regionParam, storageResource.getRegion().name());
+            params.addValue(regionParam, ((GoogleRegion) storageResource.getRegion())
```

Comment:
Is this cast necessary? `parms.addValue()` accepts `Object`.

Reply:
It's not. I actually ended up removing it in a second PR I'm working on...I'll remove it here